cowdao_cowpy/order_book/generated/model.py

```diff
@@ -89,9 +89,10 @@ class OnchainOrderData(BaseModel):
 
 
 class EthflowData(BaseModel):
-    refundTxHash: TransactionHash = Field(
+    refundTxHash: Optional[TransactionHash] = Field(
```

Comment:
openapi spec needs to include nullable = True for this result.


It doesnt!


@ribeirojose I think maybe a pr upstream is necessary into the openapi-spec?